runtime-modules/content/src/lib.rs

```diff
@@ -1535,6 +1535,43 @@ decl_module! {
             ));
         }
 
+        /// Destroy NFT
+        #[weight = 10_000_000] // TODO: adjust weight
+        pub fn destroy_nft(
+            origin,
+            actor: ContentActor<T::CuratorGroupId, T::CuratorId, T::MemberId>,
+            video_id: T::VideoId
+        ) {
+            // Ensure given video exists
+            let video = Self::ensure_video_exists(&video_id)?;
+
+            // Ensure nft is already issued
+            let nft = video.ensure_nft_is_issued::<T>()?;
+
+            // Ensure channel exists, retrieve channel owner
+            // let channel = Self::ensure_channel_exists(&video.in_channel)?;
+            // Question - Should NFT Destruction be a pausable feature?
+            // channel.ensure_feature_not_paused::<T>(PausableChannelFeature::VideoNftDestruction)?;
```

Comment:
@bedeho Should we add an enum variant `PausableChannelFeature::VideoNftDestruction` to allow moderators to prevent nft destruction on channels?

Reply:
No I don't think so, you should be able to destroy assets that are really yours, this does not harm third parties in any obvious way.

Also critically, and related I think, look at this: https://github.com/Joystream/joystream/issues/2655#issuecomment-1086058293